apps/condo/domains/common/components/containers/amplitude/TrackPageLoad.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { LogOnMount } from 'react-amplitude-hooks'
+import { AmplitudeEventType } from '@condo/domains/common/components/containers/amplitude/AmplitudeProvider'
+import { AuthorizedUserEventProperties } from '@condo/domains/common/components/containers/amplitude/AmplitudeAuthorizedUser'
+
+export enum AmplitudePageState {
+    Empty = 'Empty',
+    InProgress = 'InProgress',
+    Error = 'Error',
+    Success = 'Success',
+    AccessError = 'AccessError',
+}
+
+interface IAmplitudeLogOnMountEvent {
+    eventType: AmplitudeEventType
+    pageState?: AmplitudePageState
+    extraEventProperties?: Record<string, number | string>
+}
+
+const TrackPageLoadEvent: React.FC<IAmplitudeLogOnMountEvent> = (props) => {
```

Comment:
The name brings false understanding, where it seems, like we're mounting an event.
More clear would be to name it like `PageLoadTracker`.

Reply:
Btw, we have `PageWrapper`, why not to integrate logic into it and use it like following:

```
<PageWrapper loadTrackEvent="…" loading={loading}>
```

This way to will not modify our components tree, but will just add a property.